src/languageHandler.js

```diff
@@ -338,8 +338,9 @@ export function getCurrentLanguage() {
 
 function getLangsJson() {
     return new Promise((resolve, reject) => {
+        // LANGUAGES_FILE is a webpack compile-time define, see webpack config
         request(
-            { method: "GET", url: i18nFolder + 'languages.json' },
+            { method: "GET", url: require(LANGUAGES_FILE) },
```

Comment:
Should the React SDK fallback to the old value, in case someone is using the SDK separately? Or if there's no fallback, should we skip this work if no `LANGUAGES_FILE` is provided?

Reply:
> Looks good overall to me! grin For the unit tests, maybe you need to sprinkle the new value in `karma.conf.js`?

Ah, yes, didn't realize we also have a webpack config section there. They should have gotten fixed by falling back to i18n/languages as before, so I think it's fine.